devp2p/kademlia.py

```diff
@@ -201,6 +208,15 @@ def to_binary(x):  # left padded bit representation
                 return i - 1
         raise Exception
 
+    def get_node(self, node):
+        if node in self:
```

Comment:
this is unnecessarily looping the self.node twice. 

alternative:
`return ([n for n in self.nodes of n.id == node.id] or [None])[0]`